config/webpack.config.prod.js

```diff
@@ -251,7 +254,10 @@ module.exports = [{
   },
   externals: /^[a-z\-0-9]+$/,
   module: {
-    loaders: loaders
+    loaders: loaders.concat({
+      test: /\.css$/,
+      loader: 'ignore-loader'
+    })
   },
   plugins: [
```

Comment:
need to include Extract in plugins too 
from CRA: 
`new ExtractTextPlugin('static/css/[name].[contenthash:8].css'),`